README.md

```diff
@@ -36,7 +36,11 @@
     ```
     If you use azure instead, set `AZURE_OPENAI_ENDPOINT` and `AZURE_OPENAI_KEY`
 
-6. 🏃‍️ Run the app
+6. 🔑 Set Authentication configuration (optional)
+
+   To set up authentication, create a config file `auth.yaml` and follow the detailed instructions in the [Streamlit-Authenticator documentation](https://github.com/mkhorasani/Streamlit-Authenticator#1-creating-a-configuration-file). Note that this app does not support new user registration or password resetting.
+
+7. 🏃‍️ Run the app
```

Comment:
These steps: clone the repo, create `.env`, create `auth.yaml` need to be consolidated in one single place (e.g the `Quick start` section) since you have exactly the same information in 2 different blocks, either with `streamlit run app.py` or `docker`... users will have to do the above steps.

For this PR scope, I keep your README structure unchanged.